lib/Parser.js

```diff
@@ -12,7 +12,7 @@ var CH_LF = 10,
     EMPTY_READCB = function(n) {},
     RE_INTEGER = /^\d+$/,
     RE_PRECEDING = /^(?:(?:\*|A\d+) )|\+ ?/,
-    RE_BODYLITERAL = /BODY\[(.*)\] \{(\d+)\}$/i,
+    RE_BODYLITERAL = /BODY\[(.*)\](<[0-9]+>)* \{(\d+)\}$/i,
```

Comment:
The `*` operator in this change isn't correct I think. There is only ever one set of `<` and `>`. It should instead be a `?` to make it optional.